scripts/ingest_data.py

```diff
@@ -1,17 +1,38 @@
 import argparse
+import logging
 from house_package_saachi.data_ingestion import fetch_housing_data, load_housing_data
 
+def setup_logger(log_level, log_path=None, no_console_log=False):
+    logger = logging.getLogger()
```

Comment:
change this to `logger = logging.getLogger(__name__)`. This will ensure that the specific _module name_ shows up in the logs instead of showing `root` everywhere like this..
```
2024-09-02 11:24:52,816 - root - INFO - Loading housing data.
2024-09-02 11:25:44,340 - root - INFO - Loading housing data.
2024-09-02 11:26:24,180 - root - INFO - Loading housing data.
2024-09-02 11:26:24,203 - root - INFO - Separating target from features.
```

Reply:
Changed accordingly on ingest_data.py, train.py and score.py